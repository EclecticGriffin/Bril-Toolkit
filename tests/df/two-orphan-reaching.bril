# ARGS: reaching_defns
@main {
    v: int = const 4;
    jmp .else;
    v: int = const 2;
.somewhere:
    print v;
.nowhere:
    print v;
.else:
    print v;
}
