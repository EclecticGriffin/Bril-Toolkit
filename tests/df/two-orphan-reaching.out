

Running reaching definitions analysis on main

Function start:
 Input:

Block 1 [unlabeled]
 Input:
 Output: v_1

Block 2 [unlabeled]
 Input:
 Output: v_2

Block 3 [somewhere]
 Input: v_2
 Output: v_2

Block 4 [nowhere]
 Input: v_2
 Output: v_2

Block 5 [else]
 Input: v_1 v_2
 Output: v_1 v_2

